From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Thu, 29 Apr 2021 09:09:20 +0100
Subject: desktop file

Update the desktop file so it validates, and standardize icon name
---
 src/eln.desktop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/eln.desktop b/src/eln.desktop
index b8d51a6..188db59 100755
--- a/src/eln.desktop
+++ b/src/eln.desktop
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Name=eln
 GenericName=Electronic Lab Notebook
-Comment=Electronic Lab Notebook
 Exec=eln %f
 Terminal=false
 Type=Application
 Categories=Office
-Icon=net.danielwagenaar.eln.png
+Icon=eln
 MimeType=application/eln-book
+Keywords=science;laboratory;data analysis;screen shots;literature search;PubMed;note taking;
